modules/theme/src/themes/default/globals/site.variables

```diff
@@ -26,7 +26,9 @@
 @importGoogleFonts : false;
 
 @iconFont          : "font-awesome";
-@fontName          : -apple-system, BlinkMacSystemFont, Segoe UI, HelveticaNeue-Light, Ubuntu, Droid Sans, sans-serif, @iconFont;
+@primaryFont       : "";
```

Comment:
Is this expect to be empty?

Reply:
Yes. In the default theme, we don't need to define a primary font.